### PR TITLE
Load improvements

### PR DESCRIPTION
**Pull request by gokmen**
- [x] All `appStorage` requests queued up on client-side
- [x] Initial `JCombinedAppStorage` data injected to payload
- [x] Editor storage dependency fixed

### CODE REVIEW

File: client/ace/lib/ace.coffee [hunk 1]
```diff
@@ -12,9 +12,7 @@ settings             = require './settings'
 
 prependWithOrigin = (path) -> [location.origin, path].join '/'
 
-module.exports =
-
-class Ace extends KDView
+module.exports = class Ace extends KDView
```
Comment (sinan) [inline client/ace/lib/ace.coffee]: 👍

File: client/stack-editor/lib/editor/basestackeditorview.coffee [hunk 2]
```diff
@@ -59,7 +59,7 @@ module.exports = class BaseStackEditorView extends IDEEditorPane
       kd.utils.defer =>
         @getEditorSession().setScrollTop 0
 
-      @emit 'EditorReady'
+      @emit 'ready'
```
Comment (sinan) [inline client/stack-editor/lib/editor/basestackeditorview.coffee]: 👍

**Review by sinan — APPROVED** [review]
Looks good to me! 💯

File: client/ace/lib/ace.coffee [hunk 3]
```diff
@@ -90,11 +88,10 @@ class Ace extends KDView
 
     @hide()
 
-    @appStorage.fetchStorage (storage) -> # XXX: wtf? -og
```
Comment (usirin) [inline client/ace/lib/ace.coffee]: This change is like an answer to this `wtf` :trollface:

**Comment by usirin** [comment]
💯 Thank you.

File: client/stack-editor/lib/editor/basestackeditorview.coffee (hunk 2 above)
Comment (cihangir) [inline client/stack-editor/lib/editor/basestackeditorview.coffee]: 👍

File: workers/social/lib/social/render/scriptblock.coffee [hunk 4]
```diff
@@ -104,11 +107,11 @@ module.exports = (options = {}, callback) ->
   queue = [
 
     (fin) ->
-       socialApiCacheFn = require '../cache/socialapi'
-       socialApiCacheFn options, (err, data) ->
-         console.error 'could not get prefetched data', err  if err
-         socialapidata = data
-         fin()
+      socialApiCacheFn = require '../cache/socialapi'
```
Comment (cihangir) [inline workers/social/lib/social/render/scriptblock.coffee]: 👍

File: workers/social/lib/social/render/scriptblock.coffee [hunk 5]
```diff
@@ -138,6 +140,15 @@ module.exports = (options = {}, callback) ->
         fin()
 
     (fin) ->
+      { delegate : account } = client.connection
+      query = { accountId : account._id }
+      bongoModels.JCombinedAppStorage.one query, (err, storage) ->
```
Comment (cihangir) [inline workers/social/lib/social/render/scriptblock.coffee]: 👍

**Review by cihangir — APPROVED** [review]

**Pull request merged**